src/QueryStrategies/BatchQueryStrategies/batch_qs_base.jl

```diff
@@ -0,0 +1,130 @@
+abstract type BatchPQs <: PoolQs end
+abstract type ExtendingBatchQs <: BatchPQs end
+abstract type MultiObjectiveBatchQs <: BatchPQs end
+
+"""
+    select_batch(batch_query_strategy, query_data, label_map, candidate_indices)
+Applies batch_query_strategy to select the most useful unlabeled observations
+from query_data.
+
+label_map is used to find out which observations in query_data are still unlabeled,
```

Comment:
This should read "labels" instead of label_map

Reply:
Changing to `pools` as this is the correct parameter name of the function.